docs/source/model_design/programming_vivarium/running_simulations/index.rst

```diff
@@ -31,9 +31,207 @@ Running Simulations
+
+Running a Simulation on the Command Line
+----------------------------------------
+
+Once you've confirmed that your artifact is correct, and that your 
+:code:`model_spec.yaml` and :code:`branches/scenarios.yaml` files 
+are up to date, you're ready to run the model! 
+
+Creating an Environment
++++++++++++++++++++++++
+
+First, you will need to create an environment. This will be the same
+as creating usual environments, except when you install the project repo,
+you'll need to do a special editable install. The command looks like this :code:`pip install -e.[dev]`. Note that this is DIFFERENT than what is required for artifact building. 
+
+Additionally you'll need to install redis with :code:`conda install redis`.
+Some other packages can be helpful here as well like :code:`ipython` for
+debugging. 
+
+Generally, the project repo and redis are the only things you will need to install,
+but check with the engineers if there are updates to other vivarium
+packages you should be aware of - like :code:`vivarium_inputs`,
+:code:`vivarium_public_health`, or :code:`vivarium_gbd_access`.
+
+Simulate
+++++++++
+
+The :code:`simulate` run will run a single simulation based on the 
+:code:`model_spec.yaml` file. Since you're only running one simulation, 
+it won't look at the :code:`branches/scenarios.yaml` file at all. While 
+we mostly will need to use :code:`psimulate`, running a :code:`simulate` 
+run can be very helpful in debugging as it is much faster and will reveal 
+any bugs in the code. 
+
+Here is an example :code:`simulate` run and an explanation of the flags. 
+
+.. code-block:: bash 
+  :linenos:
+
+  $ simulate <PATH_TO_MODEL_SPEC> -vvv --pdb --proj_simscience_prod -o /mnt/team/simulation_science/pub/models/<PROJCET_NAME>/results/<MODEL_NUMBER>/ -i '<PATH_TO_ARTIFACT>' 
+
+Flags: 
+  - -vvv is for the verbosity, the vvv is standard on the team
+  - --pdb has you reach the python debugger if there are any errors
+  - --proj_simscience_prod has you run on the production project on the cluster, which has higher priority than other projects. You might need to be added to access this project! 
+  - -o is where to put the output results
+  - -i is the location of the artifact
+
+The simulate run should be relatively quick and won't take a tremendous amount 
+of data to run or store results. 
+
+Psimulate
++++++++++
+
+The :code:`psimulate` run will run multiple simulations, using information 
+from both the :code:`model_spec.yaml` and :code:`branches/scenarios.yaml` files.
+One way to think about this is that the :code:`branches/scenarios.yaml` 
+file with supersede the :code:`model_spec.yaml`, so anything 
+present there - draws count, seed count, or scenarios for example, 
+will therefore overwrite the corresponding lines in the :code:`model_spec.yaml`. 
+
+Since the :code:`psimulate` run is quite large, you'll need to start 
+by getting a large :code:`srun` going. There is pretty good documentation 
```

Comment:
Is there no `srun` required for a regular `simulate`? Or is it just that it can be smaller?

Reply:
I think you need an srun, or else you need to `simulate` locally.  But maybe it can be smaller?

Reply:
I'll add a sentence about this. Yes, you need an srun but it doesn't need to be 70 G and a ton of threads - it can be quite small.